jsonschema/_format.py

```diff
@@ -296,6 +296,8 @@ def is_css3_color(instance):
         draft6="json-pointer", raises=jsonpointer.JsonPointerException,
     )
     def is_json_pointer(instance):
+        if not isinstance(instance, str_types):
```

Comment:
This needs a test :)

Reply:
It already does - the test fail was what caused me to fix :)

Reply:
Ah cool. Hm, I thought I'd ran these after updating. Thanks for fixing.